### PR TITLE
Remove checking for gtk_3_10

### PR DESCRIPTION
**Pull request by EPashkin**
cc @GuillaumeGomez, @sdroege

### CODE REVIEW

File: src/bin/builder_basics.rs [hunk 1]
```diff
@@ -5,56 +5,40 @@
 extern crate gio;
 extern crate gtk;
 
-#[cfg(feature = "gtk_3_10")]
```
Comment (sdroege) [inline src/bin/builder_basics.rs]: Why did we do that inside the source code instead of just in Cargo.toml?! :)

**Comment by sdroege** [comment]
:+1:

File: src/bin/builder_basics.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply src/bin/builder_basics.rs]: Young we were.

**Comment by GuillaumeGomez** [comment]
Thanks!

**Comment by EPashkin** [comment]
@sdroege Mostly because we don't add files to Cargo.toml and use auto-detection instead

**Pull request merged**